doc/reference/reference_lua/box_ctl/demote.rst

```diff
@@ -0,0 +1,28 @@
+.. _box_ctl-demote:
+
+box.ctl.demote()
+=================
+
+..  function:: demote()
+
+    Revoke the leader role of the instance.
+
+    The function is used only for the :ref:`owner of the synchronous transaction queue <box_info_synchro>`.
+    In other words, it may be issued only on the leader when the elections are enabled.
```

Comment:
I think it's better to rewrite this paragraph:

The function is a no-op on anyone but the synchro queue owner. It returns immediately and does nothing.

On synchronous transaction queue owner the function works as follows:

* If `box.cfg.election_mode` is `off`: ...
* Iff `box.cfg.election_mode` is enabled in any mode ...

Reply:
Updated text